main.cpp

```diff
@@ -73,12 +24,31 @@ void testPairToNumber(
     assert(pairNumber == expectedPairNumber);
 }
 
+extern void PrintReferenceManual();
+
 int main() {
-    testNumberToPair(4, TelCoColorCoder::WHITE, TelCoColorCoder::BROWN);
-    testNumberToPair(5, TelCoColorCoder::WHITE, TelCoColorCoder::SLATE);
+    TelCoColorCoder::InitializeDefaultEnglishLabels();
+
+    // Test valid and invalid pair numbers
+    try {
+        testNumberToPair(4, TelCoColorCoder::WHITE, TelCoColorCoder::BROWN);
+        testNumberToPair(5, TelCoColorCoder::WHITE, TelCoColorCoder::SLATE);
+        testNumberToPair(26, TelCoColorCoder::RED, TelCoColorCoder::BLUE); // Invalid pair number
+    } catch (const std::out_of_range& e) {
+        std::cerr << "Error: " << e.what() << std::endl;
+    }
 
-    testPairToNumber(TelCoColorCoder::BLACK, TelCoColorCoder::ORANGE, 12);
-    testPairToNumber(TelCoColorCoder::VIOLET, TelCoColorCoder::SLATE, 25);
+    // Test valid and invalid color pair to number conversion
+    try {
+        testPairToNumber(TelCoColorCoder::BLACK, TelCoColorCoder::ORANGE, 12);
+        testPairToNumber(TelCoColorCoder::VIOLET, TelCoColorCoder::SLATE, 25);
+        testPairToNumber(static_cast<TelCoColorCoder::MajorColor>(999), TelCoColorCoder::BLUE, 999); // Invalid major color
+    } catch (const std::out_of_range& e) {
+        std::cerr << "Error: " << e.what() << std::endl;
+    }
+
+    // std::cout << "\nReference Manual:\n";
+    // PrintReferenceManual();
```

Comment:
Call to PrintReferenceManual is commented? Is it because you did not have an assert for it?